### PR TITLE
Component/ BlockStack

### PR DESCRIPTION
**Pull request by haild173094**

### CODE REVIEW

File: src/components/BlockStack/BlockStack.vue [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<template lang="pug">
+component(
+  :is="as",
+  :class="className",
+  :style="sanitizeCustomProperties(style)",
+)
+  slot
+</template>
+
+<script setup lang="ts">
```
Comment (HQCuong) [inline src/components/BlockStack/BlockStack.vue:10]: missing define slots

File: src/components/BlockStack/types.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
+import type {
```
Comment (HQCuong) [inline src/components/BlockStack/types.ts:1]: convention

**Review by HQCuong — CHANGES_REQUESTED** [review]
missing define slots, convention

File: src/components/BlockStack/BlockStack.vue [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+const props = withDefaults(defineProps<BlockStackProps>(), {
+  as: 'div',
+  reverseOrder: false,
+});
+
+defineSlots<BlockStackSlots>();
+
+const styles = useCssModule();
+
+const style: any = {
```
Comment (juzser) [inline src/components/BlockStack/BlockStack.vue:34]: Should be `computed`

**Review by juzser — CHANGES_REQUESTED** [review]

File: docs/components/BlockStack/PreviewPlaceholder.vue [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+}
+
+const props = withDefaults(defineProps<PlaceholderProps>(), {
+  height: 'auto',
+  width: 'auto',
+  label: '',
+  showBorder: false,
+});
+
+const wrapperStyle = {
```
Comment (juzser) [inline docs/components/BlockStack/PreviewPlaceholder.vue]: computed

**Review by juzser — APPROVED** [review]

**Pull request merged**